src/ApiSecurityDocumentation.js

```diff
@@ -46,12 +46,14 @@ export class ApiSecurityDocumentation extends AmfHelperMixin(LitElement) {
         font-weight: var(--arc-font-headline-font-weight);
         letter-spacing: var(--arc-font-headline-letter-spacing);
         line-height: var(--arc-font-headline-line-height);
+        color: color: var(--arc-font-headline-color);
```

Comment:
Double color definition. This won't fly ;)

Reply:
How embarrassing... Thanks for the notice, will fix it :)